### PR TITLE
get things working

### PR DESCRIPTION
**Pull request by scrogson**

### CODE REVIEW

File: lib/guardian_onetime.ex [hunk 1]
```diff
@@ -1,4 +1,4 @@
-defmodule Guardian.Token.OneTime do
+defmodule GuardianOneTime do
```
Comment (doomspork) [inline lib/guardian_onetime.ex]: I'm not for this module change.  👎  

The previous module was meant to match with how namespacing works for the existing `Guardian.Token.Jwt`

**Review by doomspork — CHANGES_REQUESTED** [review]
I'd like to see a different module name used.  Why make this change?

File: lib/guardian_onetime.ex (hunk 1 above)
Comment (yordis) [reply lib/guardian_onetime.ex]: I prefer if you keep everything under Guardian namespace as well

File: lib/guardian/token/one_time.ex [hunk 2]
```diff
@@ -10,9 +10,9 @@ defmodule Guardian.Token.OneTime do
 
   ```elixir
   defmodule MyApp.OneTimeToken do
-  use Guardian.Token.OneTime, otp_app: :my_app,
-  repo: MyApp.Repo,
-  token_table: "one_time_tokens"
+  use GuardianOneTime, otp_app: :my_app,
```
Comment (doomspork) [inline lib/guardian/token/one_time.ex]: Does this need to be changed?

File: test/support/repo.ex [hunk 3]
```diff
@@ -0,0 +1,3 @@
+defmodule GuardianOneTime.Repo do
```
Comment (doomspork) [inline test/support/repo.ex:1]: This should match the other namespace don't you think?

**Review by doomspork — CHANGES_REQUESTED** [review]

File: test/support/repo.ex (hunk 3 above)
Comment (scrogson) [reply test/support/repo.ex:1]: No, this is completely isolated to this package's test support.

File: lib/guardian/token/one_time.ex (hunk 2 above)
Comment (scrogson) [reply lib/guardian/token/one_time.ex]: Yes, good catch.

File: mix.exs [hunk 4]
```diff
@@ -36,6 +43,9 @@ defmodule GuardianOnetime.Mixfile do
     ]
```
Comment (hassox) [inline mix.exs:43]: Should we be marking postgrex as optional?

**Review by hassox — COMMENTED** [review]
Just wondering about the postgrex requirement in mix. Did we require this because of the map issue?

File: mix.exs (hunk 4 above)
Comment (scrogson) [reply mix.exs:43]: maybe...we should probably test this against postgres and mysql 5.7 and verify. Then we can make both optional.

**Pull request closed without merging**